mlperf_common/callbacks/logging.py

```diff
@@ -362,6 +362,8 @@ def __init__(self):
         self.run_n_iters = int(os.environ.get("RUN_N_ITERS", "0"))
         self.enabled = True
 
+        self.save_path = os.environ.get("STAT_CALLBACK_FNAME", f"/results/stats_seed{os.getenv('SEED', '1')}.json")
```

Comment:
I see the interface is using envvars, but can we move it to the constructor and consume envvars in the benchmark code?